doc-content/enterprise-only/project-data/create-branches-proc.adoc

```diff
@@ -0,0 +1,20 @@
+[id='create-branches-proc']
+
+= Creating branches
+You can create a new branches in {CENTRAL} and name them whatever you like. Initially, you will only have the default *master* branch. When you create a new branch for a project, you are making a copy of the project and all of its assets. You can then switch to your newly created branch and make changes to the project without impacting the original *master* branch version.
```

Comment:
... you are making a copy of the project and all of its assets... Although I understand this correctly, this might be a bit confusing for users - because there is an option of "duplicate" for each project, which makes the copy of the project. However, I am not sure how would I explain it better.
 Maybe Paulo could have some suggestions here? :)

Reply:
Also, "you can switch to your newly created branch" - later on, you mention that you are redirected to the newly created branch (which is true). Here, it sounds like you need to switch to it - I would rather say something like "When you are on the newly created branch" or anything like this to avoid the word "switch"

Reply:
I agree with @barboras7 , I would change "project and all of its assets" to "selected branch and all of its assets" or something like that.

Also there is a typo in the beginning of this line: "You can create a new branches" -> "You can create new branches".